agent/metasploit_agent.py

```diff
@@ -176,8 +178,7 @@ def _get_job_results(
                 results = job_result["result"]
                 break
             if status == "errored":
-                logger.error("Module error: %s", job_result["error"])
-                break
+                raise ValueError(f"""Module Error: {job_result["error"]}""")
```

Comment:
if we raise exception we'll block next msf modules from running as well.

Reply:
How does it fix the issue? + missing unit test